source/MRMesh/MREnumNeighbours.h

```diff
@@ -19,13 +19,20 @@ class EnumNeihbourVertices
 };
 
 /// computes Euclidean 3D distances from given start point to all neighbor vertices within given \param range
-/// and to first vertices with the distance more or equal than range
+/// and to first vertices with the distance more than range
 [[nodiscard]] MRMESH_API VertScalars computeSpaceDistances( const Mesh& mesh, const PointOnFace & start, float range );
 
 /// calculates all neighbor vertices within a given \param range
-/// and to first vertices with the distance more or equal than range
+/// and to first vertices with the distance more than range
 [[nodiscard]] MRMESH_API VertBitSet findNeighborVerts( const Mesh& mesh, const PointOnFace& start, float range );
 
+/// computes Euclidean 3D distances from given start point and collect all neighbor vertices within given \param range
+/// and to first vertices with the distance more than range or with normal opposite to the \param dir
+[[nodiscard]] MRMESH_API std::pair<VertScalars, VertBitSet> findSpaceDistancesAndRegionVisible( const Mesh& mesh, const PointOnFace& start, float range, const Vector3f& dir );
```

Comment:
not sure that "visible" is good word here

Reply:
I tried to rename it to ".CoincidingDirection", but not sure that is better